react-native-libraries.json

```
@@ -12297,5 +12297,13 @@
   {
     "githubUrl": "https://github.com/LunatiqueCoder/expo-notifee-plugin",
     "ios": true
+  },
+  {
+    "githubUrl": "https://github.com/LonelyCpp/rn-scroll-viewport-tracker",
+    "npmPkg": "rn-scroll-viewport-tracker",
+    "images": ["https://github.com/LonelyCpp/rn-scroll-viewport-tracker/blob/main/docs/viewport.png?raw=true"],
```

Hey @LonelyCpp, thanks for adding new library entry! 👍 

Just a small nit related to image, we can use the direct raw link to avoid redirects when fetching:

```suggestion
    "images": ["https://raw.githubusercontent.com/LonelyCpp/rn-scroll-viewport-tracker/refs/heads/main/docs/viewport.png"],
```

We can also skip `npmPkg` since it matches repository name.

---

- updated image link to `raw.githubusercontent.com`
- removed `npmPkg`

done 👍